splice/models.py

```diff
@@ -62,6 +62,13 @@ class AdgroupSite(db.Model):
     created_at = db.Column(db.DateTime(), nullable=False, default=datetime.utcnow)
 
 
+class BlacklistedIP(db.Model):
+    __tablename__ = "blacklisted_ips"
+
+    ip = db.Column(db.String(1024), nullable=False, primary_key=True)
```

Comment:
Nice.  Thanks.  I will enwiden it (yes, that's a word).

On Tue, May 12, 2015 at 3:41 PM, Nan Jiang <notifications@github.com> wrote:

> In splice/models.py
> <https://github.com/oyiptong/splice/pull/63#discussion_r30172107>:
>
> > @@ -62,6 +62,13 @@ class AdgroupSite(db.Model):
> >      created_at = db.Column(db.DateTime(), nullable=False, default=datetime.utcnow)
> >
> >
> > +class BlacklistedIP(db.Model):
> > +    __tablename__ = "blacklisted_ips"
> > +
> > +    ip = db.Column(db.String(1024), nullable=False, primary_key=True)
>
> Not necessarily to be limited by 16 chars. AFAIK, a lot of vendors also
> offer CRID IP ranges, i.e. 192.168.100.0/24, instead of giving a very
> wide list of consecutive IPs.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/oyiptong/splice/pull/63/files#r30172107>.
>